package.json

```diff
@@ -39,6 +39,7 @@
     "prepare": "husky install && playwright install",
     "lint:copyright": "node scripts/copyrightLinter.js",
     "createComponent": "node scripts/createComponent.js",
+    "approve": "pnpm run approve:css && pnpm run approve:react",
```

Comment:
Originally, not providing a top-level `approve` command was intentional, because sometimes the PR author is only working on CSS or only working on React. Running `approve:css` or `approve:react` makes it feel more intentional and prevents accidental approvals in the wrong workshop.

Having said that, I'm open to adding this command because I agree it can be a nice thing. We can keep an eye out for whether it causes confusions/mistakes in any PRs in the future, and revisit accordingly.